src/SFA.DAS.AssessorService.Web.Staff/Controllers/Roatp/UpdateRoatpOrganisationController.cs

```diff
@@ -158,6 +176,29 @@ public async Task<IActionResult> UpdateStatus(UpdateOrganisationStatusViewModel
             return View("~/Views/Roatp/UpdateOrganisationStatus.cshtml", model);
         }
 
+        [HttpPost]
+        public async Task<IActionResult> UpdateType(UpdateOrganisationTypeViewModel model)
+        {
+            model.OrganisationTypes = _apiClient.GetOrganisationTypes(model.ProviderTypeId).Result.Where(x=>x.Id!=0).OrderBy(x => x.Id);
```

Comment:
Thinking it would be a good idea to have the organisation type id 0 (Unassigned) as a constant so it's easy to see that we are excluding that particular type

Reply:
yeah, there's 2 places I've used it, and I've added it as a constant to the OrganisationType class and using that instead.  Much better than magic numbers